pkg/virt-handler/vm.go

```diff
@@ -642,6 +642,22 @@ func (d *VirtualMachineController) migrationOrphanedSourceNodeExecute(key string
 	return nil
 }
 
+func isMigrating(start, stop *v12.Time) bool {
```

Comment:
I've commented this on the original PR. I think we need to make sure that `vmi.Status.MigrationState` isn't `nil`.
Perhaps let's change this to something like the following? :

```suggestion
func isMigrating(vmi *v1.VirtualMachineInstance) bool {                         
                                                                                
    now := v12.Now()                                                            
                                                                                
    running := false                                                            
    if vmi.Status.MigrationState != nil {                                       
        start := vmi.Status.MigrationState.StartTimestamp                       
        stop := vmi.Status.MigrationState.EndTimestamp                          
        if start != nil && (now.After(start.Time) || now.Equal(start)) {        
            running = true                                                      
        }                                                                       
                                                                                
        if stop != nil && (now.After(stop.Time) || now.Equal(stop)) {           
            running = false                                                     
        }                                                                       
    }                                                                           
    return running                                                              
}               
```

Reply:
yeah, saw it on the original PR and I agree but I assume it will be handled there (@petrkotas ?), I just wanted to see how often the problem I was pointed to occurs with petr's fix

Reply:
OK..